scripts/diff_to_added_lines.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import sys
+import unidiff
+import os.path
+import json
+
+if len(sys.argv) != 3:
+  print >>sys.stderr, "Usage: diff_to_added_lines.py diff.patch repository_root_directory"
+  sys.exit(1)
+
+diff_file = sys.argv[1]
+repository_root = sys.argv[2]
+
+# Create a set of all the files and the specific lines within that file that are in the diff
```

Comment:
You might consider including this in the output on wrong number of arguments to save someone having to open the script to find out what it does.